spec/system/publishers/publishers_can_select_a_job_application_for_interview_spec.rb

```diff
@@ -244,27 +248,56 @@
           expect(publisher_ats_reference_request_page.timeline_titles.map(&:text)).to eq(["Marked as complete", "Marked as interviewing"])
         end
 
-        scenario "changing our mind and using TV after all" do
-          expect(publisher_ats_reference_request_page).to be_displayed
-          publisher_ats_reference_request_page.use_tv_anyway_link.click
+        context "when changing our mind and using TV after all" do
+          before do
+            publisher_ats_reference_request_page.use_tv_anyway_link.click
+          end
 
-          expect(publisher_ats_collect_references_page).to be_displayed
-          choose "Yes"
-          click_on "Save and continue"
+          context "without collecting references" do
+            before do
+              choose "No"
+              click_on "Save and continue"
+            end
 
-          expect(publisher_ats_ask_references_email_page).to be_displayed
-          # don't contact applicant
-          choose "No"
-          click_on "Save and continue"
+            it "stops immediately" do
+              expect(page).to have_current_path(pre_interview_checks_organisation_job_job_application_path(vacancy.id, job_application))
+            end
+          end
 
-          expect(publisher_ats_interviewing_page).to be_displayed
-          publisher_ats_interviewing_page.pre_interview_check_links.first.click
+          context "when collecting references" do
+            before do
+              choose "Yes"
+              click_on "Save and continue"
+            end
 
-          expect(publisher_ats_pre_interview_checks_page).to be_displayed
-          publisher_ats_pre_interview_checks_page.reference_links.first.click
+            scenario "errors" do
+              click_on "Save and continue"
+              expect(publisher_ats_pre_interview_checks_page.errors.map(&:text))
+                .to eq(["Select yes if you would like the service to email applicants that you are collecting references."])
+            end
 
-          expect(publisher_ats_reference_request_page).to be_displayed
-          expect(publisher_ats_reference_request_page.timeline_titles.map(&:text)).to eq(["Reference requested", "Marked as interviewing"])
+            scenario "not contacting applicant" do
+              choose "No"
+              click_on "Save and continue"
+
+              expect(publisher_ats_pre_interview_checks_page).to be_displayed
+              publisher_ats_pre_interview_checks_page.reference_links.first.click
+
+              expect(publisher_ats_reference_request_page).to be_displayed
+              expect(publisher_ats_reference_request_page.timeline_titles.map(&:text)).to eq(["Reference requested", "Marked as interviewing"])
+            end
+
+            scenario "contacting applicant" do
```

Comment:
The assertions in this test are identical to the previous test. Is there anything else we could assert that would show that when use selects "Yes" or "No" the value is persisted correctly? Does anything related to this show on the publisher_ats_reference_request_page?